### PR TITLE
Added nglview post. Also added embedded version of ngl itself.

### LINKED ISSUE #10: blog post on nglviewer and MDAnalysis
@dotsdl showed me very cool stuff... he really should show you, too

### PR DESCRIPTION
**Pull request by dotsdl**
Also included a PDB from nglview's datafiles set to demonstrate what we would roughly see from the widget directly in the browser.

Closes #10.

### CODE REVIEW

**Comment by dotsdl** [comment]
@jandom this is what happens when you prod me. :D

**Comment by dotsdl** [comment]
The post is fairly minimal, since the next best thing to showing the code and a basic in-browser view of what would be shown in the widget is to use something like [mybinder](http://mybinder.org/) and link to it to let people try it out immediately. If we want that I can make it happen, but didn't want to spend too much time on this.

File: _posts/2016-03-14-nglview.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+function to produce the Jupyter widget:
+
+{% highlight python %}
+w = nv.show_mdanalysis(protein)
+w
+{% endhighlight %}
+
+This will give something like this in the notebook itself:
+
+<html lang="en">
```
Comment (orbeckst) [inline _posts/2016-03-14-nglview.md]: Adding a new html/body inside the document is bad. Maybe try something like http://stackoverflow.com/questions/2754391/embed-javascript-in-markdown but I didn't have time to try.

File: _posts/2016-03-14-nglview.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+<head>
+	<meta charset="utf-8">
+</head>
+<body>
+
+	<script src="../../../../js/build/ngl.embedded.min.js"></script>
```
Comment (orbeckst) [inline _posts/2016-03-14-nglview.md]: Can we use `{{site.js}}` or similar for such paths?

**Comment by orbeckst** [comment]
Cool!
See comments---would be nice to have a documented, standard way for how to embed js in pages.

**Comment by jandom** [comment]
@dotsdl we were just having these innocent jokes with Oli last night – now I have to pick up the tab :P

File: _posts/2016-03-14-nglview.md (hunk 1 above)
Comment (dotsdl) [inline _posts/2016-03-14-nglview.md]: Actually, as far as I can tell from looking around, inserting html in this way is both valid and the way to do it with jekyll.

File: _posts/2016-03-14-nglview.md (hunk 2 above)
Comment (dotsdl) [inline _posts/2016-03-14-nglview.md]: I think so. That would certainly be less fragile.

**Comment by dotsdl** [comment]
I reverted the code blocks, and added `site.data` and `site.js` in `_config.yml`. These are now used instead of the hardcoded paths in the post.

File: _posts/2016-03-14-nglview.md (hunk 1 above)
Comment (orbeckst) [inline _posts/2016-03-14-nglview.md]: When you look at the served source you get illegal HTML, at least from quick searching it seems to be the case that nested HTML tag or nested body tags are incorrect. 

--
Oliver Beckstein
email: orbeckst@gmail.com


Am Mar 15, 2016 um 10:12 schrieb David Dotson <notifications@github.com>:

> Actually, as far as I can tell from looking around, inserting html in this way is both valid and the way to do it with jekyll.

**Comment by orbeckst** [comment]
Ok, thanks. Didn't have time to revert. 

What versions of the local gems so you have? Maybe need to update my Jekyll related gems. 

--
Oliver Beckstein
email: orbeckst@gmail.com


Am Mar 15, 2016 um 10:26 schrieb David Dotson <notifications@github.com>:

File: _posts/2016-03-14-nglview.md (hunk 1 above)
Comment (dotsdl) [inline _posts/2016-03-14-nglview.md]: Hmmm...in that case, I don't know what a good solution is. Perhaps we look to [`index.md`](https://github.com/MDAnalysis/MDAnalysis.github.io/blob/master/index.md) for inspiration?

**Comment by orbeckst** [comment]
@dotsdl can you check? I removed the html and body tags (mainly using http://stackoverflow.com/questions/2754391/embed-javascript-in-markdown as a guideline: add a newline (and dedent). Works for me (although despite gem update, GHF fenced code blocks still don;t display for me... I take your word that it will be fine on GH).

**Comment by orbeckst** [comment]
Btw, is there a way to make the nglviewer viewport dynamically sized? Right now it seems fixed 
```html
<div id="viewport" style="width:700px; height:400px;"></div>
```
and that breaks the otherwise fluid display.

**Comment by dotsdl** [comment]
@orbeckst maybe? Paging @arose. I'm a javascript noob.

**Comment by dotsdl** [comment]
@orbeckst here's the environment I'm working with. I'm not a Ruby person so I don't know what many of these are actually for, but perhaps you can spot what's different among your gems:

```
> gem list --local

*** LOCAL GEMS ***

activesupport (4.2.6)
bigdecimal (1.2.8)
colorator (0.1)
did_you_mean (1.0.0)
ffi (1.9.10)
html-pipeline (2.3.0)
i18n (0.7.0)
io-console (0.4.5)
jekyll (3.1.2)
jekyll-feed (0.4.0)
jekyll-mentions (1.0.1)
jekyll-paginate (1.1.0)
jekyll-sass-converter (1.4.0)
jekyll-sitemap (0.10.0)
jekyll-watch (1.3.1)
json (1.8.3)
kramdown (1.10.0)
liquid (3.0.6)
listen (3.0.6)
mercenary (0.3.5)
mini_portile2 (2.0.0)
minitest (5.8.3)
net-telnet (0.1.1)
nokogiri (1.6.7.2)
power_assert (0.2.6)
psych (2.0.17)
rake (10.4.2)
rb-fsevent (0.9.7)
rb-inotify (0.9.7)
rdoc (4.2.1)
rouge (1.10.1)
safe_yaml (1.0.4)
sass (3.4.21)
test-unit (3.1.5)
thread_safe (0.3.5)
tzinfo (1.2.2)
```

**Comment by arose** [comment]
Something like the following should work

```HTML
<div id="viewport" style="max-width:100%; height:400px;"></div>
```

```JavaScript
window.addEventListener( "resize", function( event ){
  stage.handleResize();
}, false );
```

**Comment by dotsdl** [comment]
Sweet! Thanks @arose . I'll give it a shot!

**Comment by dotsdl** [comment]
I think we're done here, unless there's anything else?

**Comment by orbeckst** [comment]
I'd say go live. 

--
Oliver Beckstein
email: orbeckst@gmail.com


Am Mar 15, 2016 um 19:11 schrieb David Dotson <notifications@github.com>:

**Comment by hainm** [comment]
@dotsdl 

In case you're interested: here is a fake trajectory player for website

http://arose.github.io/nglview/dev/api.html

**Comment by dotsdl** [comment]
Thanks @hainm! I'll have a look at this for future posts!

**Pull request merged**